src/main/java/org/jabref/gui/fieldeditors/KeywordsEditorViewModel.java

```diff
@@ -0,0 +1,90 @@
+package org.jabref.gui.fieldeditors;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import javax.swing.undo.UndoManager;
+
+import javafx.beans.property.ListProperty;
+import javafx.beans.property.SimpleListProperty;
+import javafx.collections.FXCollections;
+import javafx.util.StringConverter;
+
+import org.jabref.gui.autocompleter.SuggestionProvider;
+import org.jabref.gui.util.BindingsHelper;
+import org.jabref.logic.integrity.FieldCheckers;
+import org.jabref.model.entry.Keyword;
+import org.jabref.model.entry.KeywordList;
+import org.jabref.model.entry.field.Field;
+import org.jabref.preferences.PreferencesService;
+
+public class KeywordsEditorViewModel extends AbstractEditorViewModel {
+
+    private final ListProperty<Keyword> keywordListProperty;
+    private final Character keywordSeparator;
+    private final SuggestionProvider<?> suggestionProvider;
+
+    public KeywordsEditorViewModel(Field field,
+                                   SuggestionProvider<?> suggestionProvider,
+                                   FieldCheckers fieldCheckers,
+                                   PreferencesService preferencesService,
+                                   UndoManager undoManager) {
+
+        super(field, suggestionProvider, fieldCheckers, undoManager);
+
+        keywordListProperty = new SimpleListProperty<>(FXCollections.observableArrayList());
+        this.keywordSeparator = preferencesService.getBibEntryPreferences().getKeywordSeparator();
+        this.suggestionProvider = suggestionProvider;
+
+        BindingsHelper.bindContentBidirectional(
+                keywordListProperty,
+                text,
+                this::serializeKeywords,
+                this::parseKeywords);
+    }
+
+    private String serializeKeywords(List<Keyword> keywords) {
+        return KeywordList.serialize(keywords, keywordSeparator);
+    }
+
+    private List<Keyword> parseKeywords(String newText) {
+        return KeywordList.parse(newText, keywordSeparator).stream().toList();
+    }
+
+    public ListProperty<Keyword> keywordListProperty() {
+        return keywordListProperty;
+    }
+
+    public StringConverter<Keyword> getStringConverter() {
+        return new StringConverter<>() {
+            @Override
+            public String toString(Keyword keyword) {
+                if (keyword == null) {
```

Comment:
Please add a Java Comment when keyword can be null.

Reply:
I'm not sure when a keyword can be `null`, but I think it will never be `null`. I based this on these examples:

- https://github.com/JabRef/jabref/blob/94928dd8e0811e962be8c7ccf7145835dbe969a7/src/main/java/org/jabref/gui/fieldeditors/LinkedEntriesEditorViewModel.java#L42-L47

- https://github.com/dlsc-software-consulting-gmbh/GemsFX/blob/8e353ee0dfa370d37d8c43bb244070590390560a/gemsfx-demo/src/main/java/com/dlsc/gemsfx/demo/TagsFieldApp.java#L159-L166